app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala

```diff
@@ -93,7 +93,7 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   val vatUrl = s"$vatExternalUrl$vatStartPath"
 
   val agentSuspensionEnabled = getBoolean("features.enable-agent-suspension")
-  val welshToggleEnabled = getBoolean("features.enable-welsh-toggle")
+  val welshToggleEnabled = true
```

Comment:
why not remove this config?

Reply:
Yes please. I did ask about other configs too.

Reply:
done